src/common/legacy_config_opts.h

```diff
@@ -250,6 +250,8 @@ OPTION(mon_crush_min_required_version, OPT_STR)
 OPTION(mon_warn_on_crush_straw_calc_version_zero, OPT_BOOL) // warn if crush straw_calc_version==0
 OPTION(mon_warn_on_osd_down_out_interval_zero, OPT_BOOL) // warn if 'mon_osd_down_out_interval == 0'
 OPTION(mon_warn_on_cache_pools_without_hit_sets, OPT_BOOL)
+OPTION(mon_warn_on_noscrub, OPT_BOOL)
```

Comment:
Can you please drop these? As these old-fashion options are only maintained for backward compatibility or some performance concerned cases.

Reply:
deleted, i also confused with this file, thanks .